tests/test_filters.py

```diff
@@ -871,6 +872,51 @@ def test_decimate_boundary():
     assert boundary.n_points
 
 
+def test_extract_surface():
+    # create a single hexahedral cell
+    lin_pts = np.array([[-1, -1, -1], # node 0
+                        [ 1, -1, -1], # node 1
+                        [ 1,  1, -1], # node 2
+                        [-1,  1, -1], # node 3
+                        [-1, -1,  1], # node 4
+                        [ 1, -1,  1], # node 5
+                        [ 1,  1,  1], # node 6
+                        [-1,  1,  1]], np.double) # node 7
+
+    quad_pts = np.array([
+        (lin_pts[1] + lin_pts[0])/2.0,
+        (lin_pts[1] + lin_pts[2])/2.0,
+        (lin_pts[2] + lin_pts[3])/2.0,
+        (lin_pts[3] + lin_pts[0])/2.0,
+        (lin_pts[4] + lin_pts[5])/2.0,
+        (lin_pts[5] + lin_pts[6])/2.0,
+        (lin_pts[6] + lin_pts[7])/2.0,
+        (lin_pts[7] + lin_pts[4])/2.0,
+        (lin_pts[0] + lin_pts[4])/2.0,
+        (lin_pts[1] + lin_pts[5])/2.0,
+        (lin_pts[2] + lin_pts[6])/2.0,
+        (lin_pts[3] + lin_pts[7])/2.0], np.double)
+
+    # introduce a minor variation to the location of the mid-side points
+    quad_pts += np.random.random(quad_pts.shape)*0.25
+    pts = np.vstack((lin_pts, quad_pts))
+
+    cells = np.asarray([[20, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]], dtype=np.int64)
```

Comment:
116 characters, might be better to split it somewhere. First arg ends at 101 so it might be enough to split before `dtype`.

Reply:
Just copied the example as it fits on one line.  I breakout in a cold sweat every time I go over 79 characters and am physically ill past 100 as my lines start wrapping in emacs.